spec/system/passenger_management_spec.rb

```diff
@@ -3,142 +3,145 @@
 require 'rails_helper'
 
 RSpec.describe 'Passenger Management', :js do
-  context 'as an admin' do
-    before do
-      @user = create(:user, :admin)
-      @passenger = create(:passenger, name: 'Foo Bar')
-      @verifying_agency = create(:verifying_agency)
-      when_current_user_is(@user)
-    end
+  let!(:passenger) { create(:passenger, name: 'Foo Bar') }
+  let!(:verifying_agency) { create(:verifying_agency) }
+
+  context 'when the user is an admin' do
+    before { when_current_user_is :admin }
+
+    context 'when when creating a new passenger' do
+      before do
+        visit passengers_path
+        click_on 'Add New Passenger'
+      end
 
-    context 'creating a new passeger' do
-      context 'passenger creation successful' do
+      context 'when passenger creation is successful' do
         let :fill do
           fill_in 'Name', with: 'Foo Bar'
           fill_in 'Email', with: 'foobar@invalid.com'
           fill_in 'Address', with: '123 turkey lane'
           fill_in 'Phone', with: '123'
           fill_in 'Spire', with: '12345678@umass.edu'
-          fill_in 'How long will the passenger be with us?',
-                  with: 2.days.since.strftime('%Y-%m-%d')
-          select @verifying_agency.name,
-                 from: 'Which agency verifies that this passenger needs rides?'
+          fill_in 'How long will the passenger be with us?', with: 2.days.since.strftime('%Y-%m-%d')
```

Comment:
```suggestion
          fill_in 'How long will the passenger be with us?', with: 2.days.from_now.strftime('%F')
```

You use it later, might as well be consistent. Also improves readability, in my opinion (specifically `from_now`)